main.css

```diff
@@ -0,0 +1,314 @@
+
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+@import url('http://fonts.cdnfonts.com/css/lato-light');
+
+* {
+  margin: 0%;
+  padding: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0%;
+  font-family: 'Lato', sans-serif;
+  overflow: scroll;
+}
+
+.header_headlines_section {
+  background-image: url('./media/images/bg/header2.jpg');
+}
+
+.headlines {
+  padding: 12vh 6%;
+}
+
+.header_headlines_description {
+  border: 2px white solid;
+  background-color: #f8f6f6;
+  padding: 2%;
+  color: #636262;
+  font-family: 'Lato', sans-serif;
+  margin-bottom: 4%;
+  font-weight: bold;
+}
+
+.menu_icon,
+.menu_icon_about {
+  color: #7f7b7b;
+  font-size: 2.5rem;
+  font-weight: bolder;
+  cursor: pointer;
+}
+
+.mobile_header {
+  padding-top: 2%;
+  padding-left: 6%;
+}
+
+.desktop_header {
+  display: none;
+}
+
+.upper_headline {
+  font-family: "cocogoose", "monospace", "Poppins", sans-serif;
+  font-size: 3.3rem;
+  margin-top: 0%;
+  background: url('./media/images/bg/background_2.png');
+  background-position: top right;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-weight: 900;
+  line-height: 9vh;
+  margin-bottom: 5vh;
+}
+
+.lower_headline {
+  color: #fb5205;
+  font-size: 2rem;
+  margin-bottom: 0%;
+}
+
+.course-date {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 1.5rem;
+  font-weight: 500;
+  color: #706f6f;
+  margin: 0%;
+}
+
+.course-address {
+  font-family: 'Lato', sans-serif;
+  color: #878585;
+  font-size: 1.2rem;
+  font-weight: bold;
+  margin: 0%;
+}
+
+.main_program {
+  background-image: url('./media/images/bg/program-bg.jpg');
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2vh 0;
+}
+
+.main_program_title {
+  color: #f8f6f6;
+  margin: 0%;
+}
+
+.main_program_title_underline {
+  height: 2px;
+  width: 7%;
+  background-color: #fb5205;
+  margin: 1vh 0 3vh 0;
+}
+
+footer p {
+  font-size: small;
+  color: #555454;
+  font-weight: bold;
+  align-self: center;
+}
+
+.main_program_card {
+  background: rgba(255, 255, 255, 0.1);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 2%;
+  padding: 3%;
+}
+
+.main_program_card strong {
+  text-align: center;
+  color: #fb5205;
+  margin: auto 4%;
+  font-size: large;
+}
+
+.main_program_card .fas {
+  color: #f8f6f6;
+}
+
+.main_program_card p {
+  color: #f8f6f6;
+}
+
+.main_program_button {
+  background-color: #fb5205;
```

Comment:
- [ ] **Required** The font color should be used as given in the requirement. Kindly change this. Please ensure also that all font colors should be the same as the requirement at other places.